README.md

```diff
@@ -17,18 +17,23 @@ Open Service Broker.
 
 You'll need:
 
-- A running [kubernetes](https://github.com/kubernetes/kubernetes) or [openshift](https://github.com/openshift/origin/) cluster
+- A running [kubernetes](https://github.com/kubernetes/kubernetes) (or [openshift](https://github.com/openshift/origin/)) cluster
 - The [service-catalog](https://github.com/kubernetes-incubator/service-catalog)
   installed in that cluster
 
-If you're using helm to deploy this project, you'll need to have helm installed
-in the cluster.
+If you're using [helm](https://helm.sh) to deploy this project, you'll need to
+have it installed in the cluster.
 
 ## Getting started
 
 You can just clone this project and deploy it to start poking around right away:
 
 ```console
+# Make sure you're in the right directory of your GOPATH
```

Comment:
Just a suggestion but we can just replace the following commands with `go get`? And then just say `cd $GOPATH/src/github.com/pmorie/osb-starter-pack`